src/main/java/net/atos/zac/signalering/event/SignaleringEventObserver.java

```diff
@@ -102,12 +103,17 @@ private Signalering getSignaleringVoorGroup(final SignaleringEvent<?> event, fin
         return null;
     }
 
-    private Signalering getSignaleringVoorBehandelaar(final SignaleringEvent<?> event, final Zaak subject,
-            final ZaakInformatieobject detail) {
+    private @Nullable Signalering getSignaleringVoorBehandelaar(
```

Comment:
Minor thing, but I prefer to use another prefix for methods when they are building or calculating an outcome from input variables. Get I would try to use exclusively for property getters.

Reply:
I fully agree but if we are going down that road this PR would be incredibly huge since `get` is used all over the place. ;-) I suggest to leave this to another story.